gulpfile.js

```diff
@@ -263,7 +263,8 @@
 		 * Specifies the default task to run if no task is passed in.
 		 */
 		gulp.task("default", gulp.series("dev:run"));
-
+		/** Launches the application without building the files. For tsting prod builds. */
```

Comment:
Testing is misspelled.  

*nitpick* new lines in comment are inconsistent with the rest of the file.